packages/dom/src/dom/get-rectangle-from-range.js

```diff
@@ -89,7 +89,7 @@ export default function getRectangleFromRange( range ) {
 	// by adding a temporary text node with zero-width space to the range.
 	//
 	// See: https://stackoverflow.com/a/6847328/995445
-	if ( ! rect ) {
+	if ( ! rect || rect.height === 0 ) {
```

Comment:
Ranges set by Firefox might have 0 height (when it's collapsed on an empty string). This PR probably changes something where we're overriding the range set by Firefox, which is fine.